test/samples-js-extra/AddProductMissingColumnsExceptionFunction/function.json

```diff
@@ -0,0 +1,27 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "name": "req",
+      "direction": "in",
+      "type": "httpTrigger",
+      "methods": [
+        "post"
```

Comment:
I noticed that these are "post" here but only support "get" in the C# ones, a couple things :

1. Check why the tests would be passing (they are both passing right?) - the test should only do either a POST or a GET, which means I'd expect one or the other to be failing
2. Update the C# ones to be POST since that should be our preference for output bindings